webchat/components/contacts/contacts-sidenav.css

```diff
@@ -0,0 +1,20 @@
+.contacts-sidenav {
+    height: 100%;
+    width: 100%;
+    display: grid;
+    grid-template-rows: 56px calc(100% - 56px);
```

Comment:
auto instead of calc seems to be enough to correctly space it.

Reply:
If calc is replaced for auto, the sidenav bugs on mobile